src/react/HomePage.jsx

```diff
@@ -292,8 +292,9 @@ const InterfaceList = ({ interfaces, onInterfaceClick, maxShownInterfaces }) =>
       {remainingInterfaces > 0 && (
         <li>
           <Link to="/interfaces">
-            {remainingInterfaces} more installed
-            {remainingInterfaces > 1 ? 'interfaces' : 'interface'}…
```

Comment:
this is the default pattern of translating libraries. Right now we don't support i18n, but the code is thought so i can be easily translated in the future

Reply:
I changed that line because it was missing a space: `more installedinterfaces`. Using a single template string helps to highlight the spaces used in the string.
For the plural form of strings, I guess we would change the string entirely since i18n libraries such as i18next use helper functions to handle those cases: i.e. `t('remainingInterfaces', { count: remainingInterfaces })`
https://www.i18next.com/translation-function/plurals

Reply:
the translation must be compiled by a translator, not a programmer. While it's true that in this instance you can make the plural by adding an 's', it will be harder for a script to gather all the strings that need to be translated.

Reply:
I guess the missing space was introduced by the formatter, which saw the line as too long and broke it into multiple lines.
We can either use non breaking spaces, or use string templates